src/components/icon-as-button/_icon-as-button.scss

```diff
@@ -166,6 +170,33 @@ $includeHtml: false !default;
       }
     }
 
+    &--navy-blue {
+      fill: $iconAsButtonNavyBlueColor;
+
+      &:hover,
+      &:active {
+        fill: $iconAsButtonNavyBlueActiveColor;
+      }
+
+      .sg-icon-as-button__hole {
+        border-color: $iconAsButtonNavyBlueColor;
+      }
+    }
+
+
+    &--blue {
+      fill: $iconAsButtonBlueColor;
+
+      &:hover,
+      &:active {
+        fill: $iconAsButtonBlueActiveColor;
+      }
+
+      .sg-icon-as-button__hole {
+        border-color: $iconAsButtonBlueColor;
+      }
+    }
```

Comment:
this should be in a separate PR, right?

Reply:
i add mising color to icaon_as _button to have same everywhere